mesh/profiles/views.py

```diff
@@ -1,22 +1,49 @@
 from django.core.exceptions import ObjectDoesNotExist
-from django.views.decorators.csrf import csrf_exempt
 from django.http import JsonResponse
+from django.core import serializers
+from mesh.accounts.models import Account
 
 from mesh.profiles.models import Profile
 
+from django.views import View
 
-def bio_view(request):
-    if request.method == "POST":
-        data = request.POST
-        if 'accountID' not in data:
-            return JsonResponse({'error': 'Invalid request. Missing accountID field.'}, status=401)
-        if 'biography' not in data:
-            return JsonResponse({'error': 'Invalid request. Missing biography field.'}, status=400)
+class BiographyView(View):
+    """
+    View for getting an biography by accountId or updating an biography by accountID.
+    """
+    def get(self, request, account_id, *args, **kwargs):
+        """
+        Handle GET requests.
+
+        Retrieves a biography of the listed profile in JSON format.
+
+        Returns a JSON response containing biography of specified profile through id.
+        """
+        try:     
+            if(account_id is None):
+                return JsonResponse({'error': 'Invalid input. Missing account ID'}, status=401)
+            
+            profile = Profile.objects.get(accountID=account_id)
+            profileBiography = profile.biography
+            return JsonResponse({'message': profileBiography}, safe=False, status=200)
+        except Account.DoesNotExist:
+            return JsonResponse({'error': 'Invalid request. Account does not exist'}, status=404)
+        except Profile.DoesNotExist:
+            return JsonResponse({'error': 'Invalid request. Profile does not exist'}, status=402)
+        
+
+    def post(self, request, *args, **kwargs):
+        if request.method == "POST":
+            data = request.POST
+            if 'accountID' not in data:
+                return JsonResponse({'error': 'Invalid request. Missing accountID field.'}, status=401)
+            if 'biography' not in data:
+                return JsonResponse({'error': 'Invalid request. Missing biography field.'}, status=400)
+            else:
+                print(data.get('accountID') + " " + data.get('biography'))  # TODO: Save to database
```

Comment:
Same situation here. I would also use the exceptions in the exceptions folder and make new ones if you need custom error handling.

Reply:
I don't think Brad wrote this part, if I recall correctly this was covered in #206. It looks like he just moved the method written from that issue into the BiographyView class.

Reply:
Ohh

Reply:
Why is it showing up as his change?

Reply:
I moved it into the Biography view class. Previously, the code was just its own method, similarly to the get Profile picture method below in the same file. I thought it would make sense if it put the post method in with the get method I was doing (I think that makes sense?? dunno). 

Should I also change the post request to make it consistent with the get request I wrote?